ethereum/sidecar/bridge_out_client.go

```diff
@@ -0,0 +1,181 @@
+package sidecar
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/cosmos/cosmos-sdk/codec/types"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+
+	"cosmossdk.io/log"
+	sdkmath "cosmossdk.io/math"
+	bridgetypes "github.com/mezo-org/mezod/x/bridge/types"
+)
+
+// BridgeOutGrpcClient enables gRPC communication with mezod validator node needed
+// for the bridge-out process.
+type BridgeOutGrpcClient struct {
+	mutex          sync.Mutex
+	requestTimeout time.Duration
+	connection     *grpc.ClientConn
+}
+
+func NewBridgeOutGrpcClient(
+	logger log.Logger,
+	serverAddress string,
+	requestTimeout time.Duration,
+	registry types.InterfaceRegistry,
+) (*BridgeOutGrpcClient, error) {
+	connection, err := grpc.NewClient(
+		serverAddress,
+		// TODO: Consider using TLS protocol so that the Mezo node and Ethereum
+		//       sidecar can be run on separate servers.
```

Comment:
Is that necessary? I would expect those two to run on private archictecture without accessibility from the outside. This should be the responsibility of the node provider to secure this their own way I would say.

Reply:
Agree. Removed the TODO in fd27a6f7e9f5c2055268eb1aba6cae8f4bcb91c3.